cs-minibot-platform-src/src/main/java/minibot/BaseHTTPInterface.java

```diff
@@ -560,4 +565,33 @@ private static void stopXboxDriver() {
         // xboxControllerDriver == null
         // might get this request from stopXbox HTTP post
     }
+
```

Comment:
Documentation pls! 
 /**
         * [Description of the function]
         * Input: 
         * [variable name] - [what it is]
         * Output:
         * [what does it return? ]
         */

^^ follow a format similar to this